pkgs/development/python-modules/pyevmasm/default.nix

```
@@ -0,0 +1,28 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, future
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "pyevmasm";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = pname;
+    rev = version;
+    sha256 = "134q0z0dqzxzr0jw5jr98kp90kx2dl0qw9smykwxdgq555q1l6qa";
+  };
+
+  propagatedBuildInputs = [ future ];
+
+  checkPhase = "python tests/test_EVMAssembler.py";
+
+  meta = with lib; {
+    description = "Ethereum Virtual Machine (EVM) assembler and disassembler";
+    homepage = "https://github.com/trailofbits/pyevmasm";
+    license = licenses.agpl3;
```

```suggestion
    license = licenses.asl20;
```
https://github.com/crytic/pyevmasm/blob/master/LICENSE